### PR TITLE
coordinator: make all host rule tables exists in table 500

### PR DESCRIPTION
**Pull request by cyclinder**
---
name: Pull request
about: Tell us about your contribution
labels: ["pr/release/none-required"]
---

---
## Thanks for contributing !

Before submitting a pull request, make sure you read about our Contribution notice here: <https://spidernet-io.github.io/spiderpool/contributing/development/pullrequest/#need-review>

**What this PR does / why we need it**:

coordinator: 

- make all host rule tables exist in Table 500
- remove dependencies on NIC prefixes

**Which issue(s) this PR fixes**:

https://github.com/spidernet-io/spiderpool/issues/2161
https://github.com/spidernet-io/spiderpool/issues/2202

**Special notes for your reviewer**:

**make sure your commit is signed off**

### CODE REVIEW

**Comment by weizhoublue** [comment]
网卡名前缀的问题，CNI配置少了一个，应该还有涉及 文档的 修改

**Comment by cyclinder** [comment]
这个字段没有涉及到的文档

**Comment by weizhoublue** [comment]
> 这个字段没有涉及到的文档

那就是 我们有缺失  CNI 配置的文档 ？

File: test/doc/coordinator.md [hunk 1]
```diff
@@ -23,3 +23,4 @@
 | C00019  | Set mode to disabled, coordinator will not be called  | p3     |    |    |       |
 | C00020  | TunePodRoutes If false, no routing will be coordinated  | p3     |    |    |       |
 | C00021  | The table can be customized by hostRuleTable            | p3     |    |    |       |
+| C00022  | overwrite pod's interface name to a random name via multus and see if works           | p2     |    |    |       |
```
Comment (weizhoublue) [inline test/doc/coordinator.md:26]: when multiple interfaces for pod

Comment (cyclinder) [reply test/doc/coordinator.md:26]: 没必要，multus只有多网卡的时候才能改网卡名

**Comment by cyclinder** [comment]
> 那就是 我们有缺失 coordinator CNI 配置的文档 ？

还真没有。。

File: docs/reference/plugin-coordinator.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
+# Meta plugin: Coordinator Configuration
+
+Spiderpool provides a CNI meta-plugin called 'coordinator', which works after the main CNI is called, and provides the following main features:
```
Comment (weizhoublue) [inline docs/reference/plugin-coordinator.md:3]: update doc/mkdocs.yaml for this md

File: docs/reference/plugin-coordinator.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+  namespace: kube-system
+spec:
+  config: |-
+    {
+        "cniVersion": "0.3.1",
+        "name": "coordinator",
+        "plugins": [
+            {
+                "type": "macvlan",
+                "master": "ens160",
```
Comment (weizhoublue) [inline docs/reference/plugin-coordinator.md:31]: 作为 CNI 配置， 只说本 CNI 可配哪些， NetworkAttachmentDefinition  macvlan 这些 就不提了

**Pull request merged**